src/script/subman

```diff
@@ -1,18 +1,18 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
why do you think the user's environment always have python3 around?

Reply:
Because "make check" can only run if all build dependencies are satisfied (this is ensured by the fact that `run-make-check.sh` runs `install-deps.sh` and exits if the latter fails), and because Python 3 is a build dependency of Ceph.

Reply:
fair enough. but why do we need this change anyway? i assume, in on python3 only world, `/usr/bin/python` points to python3.

Reply:
That was my assumption too, but it's wrong. See https://fedoraproject.org/wiki/FinalizingFedoraSwitchtoPython3#.2Fusr.2Fbin.2Fpython

(As to "why" this is, I explained it in the recent ceph-devel ML thread "Transition to Python 3".)